From b087f80255e5d7bf2bf630c3ab1924f1d2005845 Mon Sep 17 00:00:00 2001
From: Matyas Selmeci <matyas@cs.wisc.edu>
Date: Wed, 11 Oct 2017 16:50:38 -0500
Subject: [PATCH] sw2932-first-fqan-manpage

Document the -all-fqans and -first-fqan options that were added in SOFTWARE-2771.
---
 doc/lcmaps_voms_localaccount.mod.8 | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/doc/lcmaps_voms_localaccount.mod.8 b/doc/lcmaps_voms_localaccount.mod.8
index 754aee6..aad0dd7 100644
--- a/doc/lcmaps_voms_localaccount.mod.8
+++ b/doc/lcmaps_voms_localaccount.mod.8
@@ -12,10 +12,11 @@ lcmaps_voms_localaccount.mod \- LCMAPS plugin to switch user identity based on V
 .RB [ \-\-add-primary-gid-as-secondary-gid-from-mapped-account ]
 .RB [ \-\-do-not-add-secondary-gids-from-mapped-account ]
 .RB [ \-\-add-secondary-gids-from-mapped-account ]
 .RB [ \-\-use-voms-gid | \-\-use_voms_gid | \-use_voms_gid ]
 .RB [ \-\-use-account-gid ]
+.RB [ \-all-fqans | \-first-fqan ]
 .hy
 .ad b
 .SH DESCRIPTION
 This VOMS localaccount acquisition plugin is a 'VOMS-aware' modification of the
 \fBlcmaps_localaccount.mod.8\fR plugin.
@@ -107,10 +108,26 @@ This option has the opposite effect of the option \fB\-\-use-voms-gid\fR,
 instructing the plugin to add the mapped account group information to the
 mapping result. This is currently already the default and hence this option has
 no effect.
 See also \fB\-\-use-voms-gid\fR.
 
+.TP
+.BI "\-first-fqan"
+When this option is set, this plugin will only consider the first FQAN for
+each VO when considering the mapping.  Any additional FQANs will be ignored.
+The explicit setting of this flag will override any defaults.
+This is currently the default.
+See also \fB\-all-fqans\fR.
+
+.TP
+.BI "\-all-fqans"
+When this option is set, this plugin will consider ALL FQANs present in
+the credential.  The explicit setting of this flag will override any defaults.
+This is currently NOT the default.
+See also \fB\-first-fqan\fR.
+
+
 .SH RETURN VALUES
 .TP
 .B LCMAPS_MOD_SUCCESS
 Success.
 .TP
-- 
2.6.3

